.bazelversion

```diff
@@ -1,4 +1,4 @@
-7.1.1
+7.4.1
```

Comment:
Note, upgraded just so we can test with `--incompatible_use_plus_in_repo_names` which wasn't present in Bazel 7.1